src/Core/functions.php

```diff
@@ -325,3 +326,119 @@ function deprecationWarning(string $version, string $message, int $stackFrame =
         trigger_error($message, E_USER_DEPRECATED);
     }
 }
+
+if (!function_exists('Cake\Core\toString')) {
```

Comment:
This check is not necessary, no one else should be defining this namespaced functions. Such checks are necessary only when defining global functions.

Reply:
thx. Fixed in https://github.com/cakephp/cakephp/pull/17295/commits/112b6d835647ba34d517bca7f8b162627d30b47d

Reply:
are these checks necessary for the remaining functions in functions.php?